github/repos.go

```diff
@@ -609,8 +589,7 @@ type PullRequestReviewsEnforcementUpdate struct {
 
 // AdminEnforcement represents the configuration to enforce required status checks for repository administrators.
 type AdminEnforcement struct {
-	URL     *string `json:"url,omitempty"`
```

Comment:
This is the only time I've seen the returned URL from github be used in a structure in this code.  It is hard to test too.  It was causing my test to fail while correct because URL was being compared as nil.